src/js/utils/events.js

```diff
@@ -181,6 +181,7 @@ export function fixEvent(event) {
         (doc && doc.clientTop || body && body.clientTop || 0);
     }
 
+    // NOTE: this may no longer be needed since the adoption of the 'keycode' package
```

Comment:
It could be good to make this a TODO comment to evaluate if we still need it. It's possible we do for backwards compatibility

Reply:
Yup, we probably need to keep this for backwards compatibility.

Reply:
So should I just drop the note altogether, or note that it *may* be unnecessary but is retained for backward compatibility?

Reply:
yeah, we can just remove this NOTE